app/services/api/authorize_request_service.rb

```
@@ -2,32 +2,34 @@ class Api::AuthorizeRequestService
+    raise Api::Unauthorized, "Account or password is invalid" if !author_resource || not_admin_resource
   end
 
   def verify_expiration
     raise JWT::ExpiredSignature if decoded_access_token[:expired_at].to_i < Time.current.to_i
   end
+
+  def not_admin_resource
+    resource_class == "Admin" && !author_resource.admin?
+  end
+
+  def decoded_access_token
```

RuboCop (1.73)

[warning] Method `Api::AuthorizeRequestService#decoded_access_token` is defined at both app/services/api/authorize_request_service.rb:15 and app/services/api/authorize_request_service.rb:30.

(Lint/DuplicateMethods)